deployer/src/helpers.py

```diff
@@ -67,8 +67,9 @@ def make_request(endpoint, type=None, data=None, username=None, password=None,
                               data=data)
 
         if r.status_code // 100 != 2:
-            print("ISSUE for POST request : " + url + " with params: " + str(
-                data))
+            print(
```

Comment:
Cool. But why is it a fix?

OK, sorry, let me be more direct on this. Saying "fix" in a commit title is of little value, essentially because everything is a fix (which is literally true) or nothing is a fix (which is hence induced by logic). When a code modification is introduced, this modification (hopefully) changes the current behavior of the product modified - yet only time can tell whether it's an improvement or not.

We can definitely say that a modification is being made to change from one behavior (possibly undesirable) to another. That is a fact. But qualifying the change as a "fix" is either useless (as redundant) or presumptuous. How can we know the modification proposed won't introduce another problem later? We generally can't really. How confident are we, that the behavior observed will be changed accordingly to our expectations? Same, we aren't 100% confident. And it's OK. We are (should be) used to the idea of introducing code with bugs - we try to avoid them, but they snake into our code.

In this particular instance, isn't the modification proposed just changing from one style to another? isn't the current code working? in this case, why are we saying it is a fix? The patch is just moving away from an implementation to another. Does it fix anything? Style, possibly. But then again what is not a fix?

Just say what you are doing in a commit title. Here, only one file is being effectively modified, so the scope of the modification is the file. What is being done? well, "format" is preferred over explicit calls to "+" and "str". That is what matters the most. Anything else is an interpretation, and should hence go to the commit body. The rationale being that we aren't interested in interpretation or thoughts or intent when investigating a problem in production, precisely because the modification has an side-effect which was unexpected at the time of the change (... unless we are evil people).

Reply:
Makes sense, thanks for sharing. Let's say it is a fix since it was not made the right way before :)

Reply:
It does not add any value but it handle strings the right way

Reply:
> Let's say it is a fix since it was not made the right way before

I am unconvinced.

Let me be more clear about it: please remove the term "fix" from the commit message - as you are writing yourself, the term itself does not bring any value, so it's noise on the first line of the commit message - and please refrain from using it in the future on the first line - possibly keep it, but then only for the commit message body.

Reply:
Alright will do, The fix here was referring to a type of commit message as expressed by the [angular contributing convention](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#type)